fabric-gametest-api-v1/build.gradle

```
@@ -2,6 +2,17 @@ version = getSubprojectVersion(project)
 
 loom {
 	accessWidenerPath = file("src/main/resources/fabric-gametest-api-v1.accesswidener")
+
+	runs {
+		testmodClient {
+			client()
+			name = "Testmod Client"
+			vmArg "-Dfabric-api.gametest.structures.output-dir=${file("src/testmod/resources/data/fabric-gametest-api-v1-testmod/gametest/structures")}"
+
+			ideConfigGenerated = true
```

I think this is fine to have, just dont think it needs to be added to the IDE runs.

```suggestion
			ideConfigGenerated = false
```